sova/llm/providers/anthropic_api.py

```
@@ -0,0 +1,229 @@
+            raise RuntimeError(f"Anthropic streaming error: {_sanitize_error(error)}") from error
+
+    async def check_available(self) -> tuple[bool, str]:
+        if not _HAS_ANTHROPIC:
+            return False, "anthropic SDK is not installed (pip install sova[anthropic])"
+        key = os.environ.get("ANTHROPIC_API_KEY", "").strip()
+        if not key:
+            return False, "ANTHROPIC_API_KEY environment variable is not set"
+        try:
+            client = self._get_client()
+            await client.models.list(limit=1)
+            return True, f"anthropic SDK {anthropic.__version__}"
```

Ruff (0.16.2)

[warning] Consider moving this statement to an `else` block

(TRY300)